pkg/kube/pod.go

```diff
@@ -235,37 +236,68 @@ func GetPodLogs(ctx context.Context, cli kubernetes.Interface, namespace, name s
 	return string(bytes), nil
 }
 
+// getErrorWithLogs fetches logs from pod and constructs error containing last ten lines of log and specified error message
+func getErrorWithLogs(ctx context.Context, cli kubernetes.Interface, namespace, podName string, err error, errorMessage string) error {
```

Comment:
Should this be reversed? `getLogsWithError`?

Reply:
This function is called when we definitely know that something wrong happened, and we would like to embed log to error message, here is how I constructed the function name - "get error containing logs" => `getErrorWithLogs`. Is it wrong ?

Reply:
Not a blocker and can be done in a follow up PR but how about `getErrorFromLogs`?

Reply:
agree, thank you for suggestion !